.github/CODEOWNERS

```diff
@@ -1 +1 @@
-@e-conomic/platform
```

Comment:
Why is the * needed?

Reply:
Because we want to get added as reviewers on any file changed

Reply:
I have the same question too

Reply:
@mihaitabamburic 
Then why are we removing `*` in other reps?